tests/in/workgroupUniformLoad.wgsl

```diff
@@ -0,0 +1,13 @@
+let SIZE: u32 = 128u;
+
+@group(0) @binding(0)
+var<storage,read> arr_i32: array<i32, SIZE>;
+
+@compute @workgroup_size(4)
+fn test_workgroupUniformLoad(@builtin(workgroup_id) workgroup_id: vec3<u32>) {
+    let x = &arr_i32[workgroup_id.x];
+    let val = workgroupUniformLoad(x);
```

Comment:
This test case is incorrect - the argument to `workgroupUniformLoad` must be uniform, but the pointer you've constructed here is definitely not uniform.

Reply:
Can you clarify how this is non uniform?

The argument to the field access is uniform, the global is uniform, so both arguments to the indexing operation are uniform. I'm not sure which operation here introduces non-uniformity.

Reply:
I'm sorry - the patch says `workgroup_id`, which is uniform, but I somehow read `local_invocation_id`. The test is fine.